src/xmpp/iq.c

```diff
@@ -241,9 +245,6 @@ iq_handlers_init(void)
         xmpp_timed_handler_add(conn, _autoping_timed_send, millis, ctx);
     }
 
-    iq_handlers_clear();
```

Comment:
There is no difference where to call these functions within `iq_handlers_init()`. I would prefer to see definitions before the rest of code:
```C
iq_handlers_init(void)
{
    xmpp_conn_t * const conn = connection_get_conn();
    xmpp_ctx_t * const ctx = connection_get_ctx();
    /* The rest of the code */
```